lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -100,6 +100,20 @@
                                 options: fieldsData.country.options,
                                 persistChanges: true
                             })
+                        },
+                        {
+                            view: new  AccountSettingsFieldViews.DropdownFieldView({
+                                model: userPreferencesModel,
+                                required: true,
+                                title: gettext('Time Zone'),
+                                valueAttribute: 'time_zone',
+                                helpMessage: gettext(
+                                    'The time zone you currently reside in. ' +
+                                    'This is used to calculate course start/end times.'
```

Comment:
Sorry. I haven't pushed the new code yet. That old code is for the first image. I updated it to look like the second image. I was just waiting to see if there were any more thoughts on list filtering so that i could push it all at once.